src/main.rs

```diff
@@ -61,6 +61,9 @@ struct Cli {
 	/// Enable kittythink mode (thought bubble)
 	#[clap(long, short)]
 	think: bool,
+	/// use custom colors
+	#[arg(long, short, num_args(2))]
+	colors: Option<Vec<u8>>,
```

Comment:
The issue with this option is it is pretty vague how to use it without knowing the underlying code. Maybe a longer docstring could help?
```suggestion
	/// Use custom colors. The first colors the message, the second colors the cat
	#[arg(long, short, num_args(2))]
	colors: Option<Vec<u8>>,
```